File: android/src/main/java/com/capacitor/community/klarna/kco/KlarnaKcoConfig.java
package com.capacitor.community.klarna.kco;
import android.content.Context;
import android.graphics.Color;

import com.getcapacitor.Plugin;

public class KlarnaKcoConfig {
    private String androidReturnUrl = "";
    private Boolean handleValidationErrors = false;
    private Boolean handleEPM = false;

    public String getAndroidReturnUrl() {
        return androidReturnUrl;
    }

    public void setAndroidReturnUrl(String androidReturnUrl) {
        this.androidReturnUrl = androidReturnUrl;
    }

    public Boolean getHandleValidationErrors() {
        return handleValidationErrors;
    }

    public Boolean getHandleEPM() {
        return handleEPM;
    }

    public void setHandleValidationErrors(Boolean handleValidationErrors) {
        this.handleValidationErrors = handleValidationErrors;
    }

    public void setHandleEPM(Boolean handleEPM) {
        this.handleEPM = handleEPM;
    }
}

File: android/src/main/java/com/capacitor/community/klarna/kco/KlarnaKcoConfig.kt
package com.capacitor.community.klarna.kco

import com.klarna.mobile.sdk.api.KlarnaEnvironment
import com.klarna.mobile.sdk.api.KlarnaLoggingLevel
import com.klarna.mobile.sdk.api.KlarnaRegion
import com.klarna.mobile.sdk.api.KlarnaTheme

class KlarnaKcoConfig {
    var androidReturnUrl = ""
    var handleValidationErrors = false
    var handleEPM = false
    var loggingLevel = KlarnaLoggingLevel.Off
    var region = KlarnaRegion.EU
    var environment = KlarnaEnvironment.PLAYGROUND
    var theme = KlarnaTheme.LIGHT
}